src/examples/_03_solabi.rs

```diff
@@ -0,0 +1,44 @@
+use ethrpc::{eth, types::*};
+use futures::future::join_all;
+use hex_literal::hex;
+use solabi::{
+    decode::{self},
+    Address,
+};
+use std::{borrow::Cow, error::Error};
+
```

Comment:
I would do it this way:

```rust
const NAME: FunctionEncoder<(), (String,)> =
    FunctionEncoder::new(solabi::selector!("name()"));

// To build the call:
TransactionCall {
    to: Some(address),
    input: Some(NAME.encode_params(&())),
    ..Default::default()
};

// To decode the return bytes
NAME.decode_returns(&bytes);
```

Reply:
Funny. I just landed on that. It's right at the top of the docs.

Reply:
RTFM 😛 

I actually forgot I had added an example of this to the docs.

Reply:
The part of the docs that is unclear is that the stuff shown there is what you should use to transform your function encoder into TransactionCall (part of a different library).

Reply:
I guess `let call` can be replaced with `let calldata`? Otherwise it felt pretty self-explanatory to me. Feel free to propose documentation improvements!